src/helpers/firebaseHelpers.js

```diff
@@ -0,0 +1,6 @@
+export const parseData = (querySnapshot) =>
+  querySnapshot.docs.reduce((acc, doc) => {
+    const { name, date } = doc.data();
+    const id = doc.id;
+    return [...acc, { id, name, date }];
```

Comment:
Nit: This can also be simplified to:
```suggestion
    return [...acc, { id: doc.id, name, date }];
```